### PR TITLE
CLDR-17046 Generate cldr-json 44.0.0-BETA2

### PR DESCRIPTION
**Pull request by btangmu**
- This results from running cldr-generate-json.sh, and other steps described in UPDATING.md

### CODE REVIEW

**Comment by btangmu** [comment]
I just noticed that UPDATING.md doesn't mention this essential step, that is, to make a PR and get it merged -- a pre-requisite for "Create a GitHub release"

**Comment by btangmu** [comment]
@srl295 should I add any other reviewers? And, should we clarify what reviewers are expected to do, given the difficulty of viewing the files in the github interface?

**Comment by srl295** [comment]
> @srl295 should I add any other reviewers? And, should we clarify what reviewers are expected to do, given the difficulty of viewing the files in the github interface?

Anything you think would be helpful would be fine to add.  

@manishearth ?

File: cldr-json/cldr-annotations-derived-full/annotationsDerived/so/annotations.json [hunk 1]
```diff
@@ -15048,7 +15048,7 @@
           "nin niqaaban",
           "nin niqaamaban",
           "nin niqaamaban: jidh fudud",
-          "niqaab\\\\"
+          "niqaab\\"
```
Comment (srl295) [inline cldr-json/cldr-annotations-derived-full/annotationsDerived/so/annotations.json:15051]: this is due to CLDR-16662 being fixed.

**Review by srl295 — APPROVED** [review]
rubber stamp LGTM

**Review by Manishearth — APPROVED** [review]
also lgtm, we'll work on ICU4X's cldr-json import later and let you know if there are any problems there. probably not soon

**Pull request merged**